### PR TITLE
adjust and expose typography

### PR DESCRIPTION
**Pull request by solomontang**
# Feature
- Expose `<Typography />` component
- Adjust typography theme to match OpenLattice designs
  - h5, h6, subtitle1, subtitle2 variants are identical due to lack of specifications
  - body1 and body2 variants are identical due to lack of specifications
  - https://www.figma.com/file/qZhKXPTjvmcPsbvYOnJMBg/Typography-Library?node-id=32%3A7
image

### CODE REVIEW

File: src/theme/typography.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+// override mui default
+// https://material-ui.com/customization/default-theme/
+
+const typography = {
+  htmlFontSize: 16,
```
Comment (UnsungHero97) [inline src/theme/typography.js:5]: What does this do?

File: src/theme/typography.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  fontFamily: "'Inter', 'Arial', sans-serif",
+  fontSize: 16,
+  fontWeightLight: 300,
+  fontWeightRegular: 400,
+  fontWeightMedium: 500,
+  fontWeightBold: 600,
+  h1: {
+    fontFamily: "'Inter', 'Arial', sans-serif",
```
Comment (UnsungHero97) [inline src/theme/typography.js:13]: How about moving `fontFamily: "'Inter', 'Arial', sans-serif",` into a variable?

```
const fontFamily = "'Inter', 'Arial', sans-serif";

h1: {
  fontFamily,
  ...
},
h2: {
  fontFamily,
  ...
},
...
```

File: src/theme/typography.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    fontWeight: 600,
+    fontSize: '1.75rem', // 28px
+    lineHeight: 1.5,
```
Comment (UnsungHero97) [inline src/theme/typography.js:16]: Same with `lineHeight`.

```
const lineHeight = 1.5;
```

Comment (solomontang) [reply src/theme/typography.js:16]: I've left it for now as it's not uncommon to have different line heights for each h1 - h6

File: src/theme/typography.js (hunk 1 above)
Comment (solomontang) [reply src/theme/typography.js:5]: i think it sets the font-size in the html tag to 16px, but unsure. I am just using the template provided by the default theme
https://material-ui.com/customization/default-theme/

**Pull request merged**